scripts/publish-release.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+source ./scripts/enable-git-ssh.sh
+set-git-ssh
+
+PACKAGE=$1
+echo $PACKAGE
+
+case $PACKAGE in
+    tester|transformer|as)
+        if [[ "$PACKAGE" == "as" ]]; then
+            NPM_PACKAGE=@massalabs/as
+        else
+            NPM_PACKAGE=@massalabs/as-$PACKAGE
+            pushd $PACKAGE
+        fi
+        # remove husky install script
+        npm pkg delete scripts.prepare
+        npm publish --access public
+        ;;
+    *)
+        echo "Unknown package $PACKAGE"
+        exit 1;;
+esac
```

Comment:
You can improve readibility by removing the nested if:
```suggestion
NPM_PACKAGE=@massalabs/as
case $PACKAGE in
    tester|transformer)
        NPM_PACKAGE=$NPM_PACKAGE-$PACKAGE 
        cd ..
        ;&
    as) 
        npm pkg delete scripts.prepare # remove husky install script
        npm publish --access public
        ;;
    *)
        echo "Unknown package $PACKAGE"
        exit 1
        ;;
esac
```

More info on ;& [here](https://www.gnu.org/software/bash/manual/html_node/Conditional-Constructs.html#index-case)

Reply:
nice!